test-kuttl/e2e/simple-rclone/08-waitfor-sync-source.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+set -e -o pipefail
+
+while [[ $(kubectl -n "$NAMESPACE" get ReplicationSource/source -o jsonpath="{.status.conditions[0].lastTransitionTime}}") == "<no value>" ]]; do
```

Comment:
What about using `.status.lastSyncTime` here?
I'm concerned that the Reconciled contition will be added before the Job actually completes